server/services/interventionsService.ts

```diff
@@ -447,4 +453,19 @@ export default class InterventionsService {
       data: { ...appointmentUpdate },
     })) as ActionPlanAppointment
   }
+
+  async recordAppointmentAttendance(
```

Comment:
I would love it if we could add the interventions service mocks at the same time as we add new endpoints; it saves time and context switching when we subsequently need the mocks during feature development. Would you be able to do that please?

Reply:
Have added the mocks (and also the ones we use in integration tests)